packages/react-storefront/src/Lazy.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright © 2017-2019 Moov Corporation.  All rights reserved.
+ */
+import React, { useState } from 'react'
+import ReactVisibilitySensor from 'react-visibility-sensor'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+export const styles = () => ({
+  container: {
```

Comment:
```suggestion
  root: {
```
Our convention is to use `root` for the classes applied to the root element.